P3481.bs

```diff
@@ -215,41 +290,76 @@ auto bulk_chunked_seq(auto prev, auto size, auto f);
+}
+```
+
+## Conflicting execution policies ## {#design.conflict.policies}
+
+For [[P2500R2]] design we previously agreed that there might be an execution policy attached to a `scheduler`. The
+question that comes to mind is what should we do if we have execution policy passed via `bulk` parameters that conflicts with
+execution policy attached to a scheduler?
+
+This is another area to explore. It's quite obvious that execution policy passed to `bulk` describes possible parallelization
+ways of `bulk` callable object, while it's not 100% obvious what execution policy attached to a scheduler means outside of
+[[P2500R2]] proposal.
+
+We might choose different strategies (not all of them are mutually exclusive):
+- Reconsider the decisions we made for [[P2500R2]] by stopping attaching policies to schedulers.
+- Reduce policies: choose the most conservative one. For example, for the passed policy `par` and attached policy `seq` we
+    reduce it the resulting policy to `seq` because it's the most conservative one.
+    - For "peer" conflicting policies: `par` and `unseq` we might either want to reduce it to `seq` or might want to give a
+        compile-time error.
+- Give a compile-time error right away for any pair of conflicting policies.
 
-We would want to relax this over-specification of the algorithm and specify the minimal properties that need to hold for the implementation.
-We would want to specify that the functor needs to be called for the entire range, but leave the details to the implementation.
-This allows the implementation to fine-tune the algorithm and make it more performant for the envisioned usage/hardware.
+Anyway, exploration of this problem is out of scope of this paper.
 
-Such a relaxed specification should allow:
--	Calling the given functor concurrently.
--	Making decay copies of the values produced by the previous sender (if the values produced are copyable).
--	Handling cancellation inside the `bulk` algorithm.
+## Expectations on customizations ## {#design.expectations}
 
-With this change we also require that the given functor be copy-constructible.
-This will make `bulk` require the same type of function as a `for_each` with an execution policy.
+The current specification of `bulk` in [[exec.bulk]] doesn't define any constraints/expectations for the customization of the algorithm.
+The default implementation (a serial version of `bulk`) is expected to be very different that its customizations.
 
-The derided direction is to specify that `bulk` can react to cancellation requests, but leave the details of how cancellation is implemented to be unspecified.
-One implementation might choose to check for cancellation for every iteration, while others may want to check for cancellation on processing a batch of items.
+Having customizations in mind, we need to define the minimal expectations.
+We propose the following:
+- Rely on `f` to be copy-constructible. That means, that `bulk` API shall have a constrain for `f`.
```

Comment:
I think it's still not an behavior of customization. It's a property of `f`, which customizations would rely on. Try to rewrite in that way.